torch/csrc/autograd/variable.cpp

```diff
@@ -322,6 +320,12 @@ namespace impl {
     return static_cast<AutogradMeta*>(self.unsafeGetTensorImpl()->autograd_meta());
   }
 
+  DifferentiableViewMeta* get_view_autograd_meta(const Variable& self) {
```

Comment:
nit: Would we want to use `Tensor` here for our variables names here? or do we keep `Variable` for consistency.

Reply:
Yes we have a `using Variable = at::Tensor` at the top of the header.
We should remove that and rename all of them I think yes.